site/docs/uk/guide/reactions.md

```diff
@@ -0,0 +1,171 @@
+# Реакції
+
+Боти можуть працювати з реакціями на повідомлення.
+Існує два типи реакцій: звичайні емодзі та користувацькі емодзі.
+
+## Реагування на повідомлення
+
+Боти можуть додавати одну емодзі-реакцію на повідомлення.
+
+У тих же випадках боти також можуть реагувати на повідомлення користувацькими емодзі. хоч боти і не можуть мати [Telegram Premium](https://telegram.org/faq?setln=uk).
+Коли преміум користувач додає до повідомлення користувацьку емодзі-реакцію, боти можуть згодом додати таку саму реакцію до цього повідомлення.
+Крім того, якщо адміністратор чату явно дозволяє використовувати користувацькі емодзі, вони також можуть бути використані ботами в цьому чаті.
+
+Ось як ви можете реагувати на повідомлення:
+
+```ts
+// Використовуємо `ctx.react` для реакції на поточне повідомлення.
+bot.command("start", (ctx) => ctx.react("😍"));
+bot.on("message", (ctx) => ctx.react("👍"));
+
+// Використовуємо `ctx.api.setMessageReaction` для реакції деінде.
+bot.on("message", async (ctx) => {
+  await ctx.api.setMessageReaction(chat_id, message_id, "🎉");
+});
+
+// Використовуємо `bot.api.setMessageReaction` поза обробниками.
+await bot.api.setMessageReaction(chat_id, message_id, "💯");
+```
+
+Як завжди, TypeScript забезпечить автодоповнення для емодзі, які ви можете використовувати.
+Список доступних емодзі-реакцій можна знайти [тут](https://core.telegram.org/bots/api#reactiontypeemoji).
+
+::: tip Плагін емодзі
+Програмувати з емодзі може бути доволі погано.
+Не всі системи можуть правильно відображати ваш вихідний код.
+Крім того, дратує постійно копіювати їх з різних місць.
+
+Дозвольте [плагіну емодзі](../plugins/emoji.md#useful-data-for-reactions) допомогти вам!
+:::
+
+Звісно, якщо ваш бот обробляє користувацькі емодзі-реакції від преміум користувачів, ви можете прослуховувати і їх.
+
+```ts
+bot.reaction(
+  { type: "custom_emoji", custom_emoji_id: "рядок-ідентифікатор" },
```

Comment:
I don't think emoji identifiers have non-Latin characters. Do you still want to keep it this way for explanatory purposes?

Reply:
```suggestion
  { type: "custom_emoji", custom_emoji_id: "identifer-string" },
```

Reply:
btw, there is a typo in English

Reply:
damn